src/nidm/experiment/Utils.py

```diff
@@ -1792,7 +1792,7 @@ def map_variables_to_terms(
                         column_to_terms[current_tuple]["responseOptions"]["unitCode"],
                     )
 
-                if "isAbout" in json_map[json_key[0]]:
+                if "isAbout" in json_map[json_key[0]].keys():
```

Comment:
shouldn't be necessary if that is a `dict`

Reply:
It is a list of dictionaries for multiple entries or single entries.  We were testing on an older version of the json sidecar files.  All isAbout entries in either BIDS-compliant JSON or NIDM-compliant JSON-LD sidecar files use a list of dictionaries now.